modules/Bio/EnsEMBL/Variation/BaseVariationFeatureOverlapAllele.pm

```diff
@@ -410,16 +410,27 @@ sub _bvf_preds {
   
   my $is_sv = $bvf->isa('Bio::EnsEMBL::Variation::StructuralVariationFeature') ? 1 : 0;
   $self->_update_preds($bvf_preds, 'sv', $is_sv, \$pred_digest);
-  
+
   # use SO term to determine class
   if($is_sv) {
     my $class_SO_term = $bvf->class_SO_term;
 
-    if($class_SO_term =~ /deletion/) {
+    my $is_CNV = $class_SO_term eq "copy_number_variation";
+    my $is_CNV_deletion  = 0;
+    my $is_CNV_insertion = 0;
+    if ($is_CNV) {
+      my $support_vars  = $bvf->structural_variation->get_all_SupportingStructuralVariants;
```

Comment:
While testing VEP, the job failed when trying to fetch the supporting SVs.
Before fetching the supporting SVs you could check if `$bvf->structural_variation` is defined.